projects/crypto-currency/destemidos/App/src/pages/Influencers/InfluencersPosts.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Card from '@material-ui/core/Card'
+import CardHeader from '@material-ui/core/CardHeader'
+import CardMedia from '@material-ui/core/CardMedia'
+import CardContent from '@material-ui/core/CardContent'
+import CardActions from '@material-ui/core/CardActions'
+import Avatar from '@material-ui/core/Avatar'
+import IconButton from '@material-ui/core/IconButton'
+import Typography from '@material-ui/core/Typography'
+import FavoriteIcon from '@material-ui/icons/Favorite'
+import ShareIcon from '@material-ui/icons/Share'
+import useStyles from './Style/styles'
+
+export default function RecipeReviewCard({banner_image, profile_image, rank, twitter_screen_name, display_name}) {
```

Comment:
Poderia adicionar espaço no começo e no final das chaves?
```suggestion
export default function RecipeReviewCard({ banner_image, profile_image, rank, twitter_screen_name, display_name }) {
```

Reply:
adicionado o espaço